sphinx/ext/autodoc/directive.py

```diff
@@ -49,19 +49,25 @@ def __getitem__(self, key):
 class DocumenterBridge:
     """A parameters container for Documenters."""
 
-    def __init__(self, env, reporter, options, lineno):
-        # type: (BuildEnvironment, Reporter, Options, int) -> None
+    def __init__(self, env, reporter, options, lineno, state):
```

Comment:
`DocumenterBridge` is a private, not public API class. But this must be a breaking change. So it would be better to make `state` argument optional.